src/plugin.js

```diff
@@ -406,10 +439,11 @@ videojs.HlsHandler.prototype.setupSourceBuffer_ = function() {
  * Seek to the latest media position if this is a live video and the
  * player and video are loaded and initialized.
  */
-videojs.HlsHandler.prototype.setupFirstPlay = function() {
-  var seekable, media;
-  media = this.playlists.media();
+HlsHandler.prototype.setupFirstPlay = function() {
+  let seekable;
+  let media;
 
+  media = this.playlists.media();
```

Comment:
Might as well make this assignment part of the media variable's declaration above.